test/protocol-tests-core/src/main/java/software/amazon/awssdk/protocol/ProtocolTestSuiteLoader.java

```diff
@@ -52,7 +52,8 @@ private Stream<? extends TestCase> loadTestCases(String testCase) {
         try {
             List<TestCase> testCases = MAPPER
                     .readValue(getClass().getResource(RESOURCE_PREFIX + testCase), new ListTypeReference());
-            return testCases.stream();
+            return testCases.stream()
+                .filter(t -> t.getIgnoreReason() == null);
```

Comment:
What is this used for?

Reply:
> What is this used for?

Nothing, I will remove it. The idea was to add other protocols from Smithy that are known to not-pass, given that those require specific semantics, e.g., handling of default values, but I instead I ended up just filtering them out. Not sure if there's value on having them here annotated as ignored with the reason why, but we can add it back if needed.